do-dont/doc/do-dont.md

```diff
@@ -124,15 +124,57 @@ const customIcon = svg`<svg
   />
 </svg>`;
 
-export const do_custom = () => html` <dockit-do
-  .color=${'#fcba03'}
-  .icon=${customIcon}
-  .label=${'HINTS'}
->
-  <ul slot="instructions">
-    <li>Lorem ipsum dolor sit amet, consectetur adipiscing elit</li>
-    <li>Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris</li>
-    <li>Duis aute irure dolor in reprehenderit in voluptate velit</li>
-  </ul>
-</dockit-do>`;
+export const do_custom = () => html`<style>
+    dockit-do.custom {
+      --dockit-do-color: #fcba03;
+    }
+  </style>
+  <dockit-do class="custom" .icon=${customIcon} .label=${'HINTS'}>
```

Comment:
We can't rely on the fact that this will take Lit template as input. Also the customer may be using different Lit version and this will create headache.

For `dockit-do` or any component in dockit-core to be used in any situation of a doc, it needs to use `attributes` or `slots` as much as possible so they can be laydown into the markdown without a `preview-story`. Because preview-story is very specific to backlight and it's not as readeable as just HTML.

Lit already treats properties as attributes too automatically so label is ok.

Reply:
This also means that all demos, if possible should be done with `HTML`  and not Lit templates. Because otherwise we are demoing examples for Lit users only.